src/library/browse/browsefeature.cpp

```diff
@@ -68,8 +68,15 @@ BrowseFeature::BrowseFeature(QObject* parent,
     QFileInfoList drives = QDir::drives();
     // show drive letters
     foreach (QFileInfo drive, drives) {
+        // Using drive.filePath() instead of drive.canonicalPath() as it
+        // delay the startup too much if there is a network share mounted
+        // (drive letter assigned) but unavailable
+        QString cp = drive.filePath();
```

Comment:
use a more descriptive variable name -- display_path is fine.  (I'm not sure what the difference between how the canonical path and filepath are used in the tree item.  Can you explain that?)